### PR TITLE
Added style and colors to the printer output, small adjustment to monitor colors

### PR DESCRIPTION
**Pull request by wjhoward**
1st pass at adding style and colors to the printer view. Related to issue: https://github.com/hatoo/oha/issues/64
I've tried to abstract the style and colors configuration into the `StyleScheme` struct.
I also made a minor adjustment to the monitor colors to make it consistent with the printer colors.

Code respects the `--disable-color` flag (but maintains the style changes, e.g. bold / underlined).
I used the `crossterm` style system which I believe tui uses, so this should be compatible with the same terminals the current code is compatible with.

The color style includes thresholds for certain values:
- **success rate**: 100 = green, > 99 = yellow, < 99 = red
- **response latency**: < 0.3 = green, 0.3 - 0.8 = yellow, > 0.8 = red
- **HTTP status**: success = green, client error = yellow, server error = red

Here's an example of how the printer view looks:
image

### CODE REVIEW

File: src/printer.rs [hunk 1]
```diff
@@ -4,11 +4,90 @@ use crate::histogram::histogram;
...
 use std::io::Write;
 use std::time::Duration;
 use std::time::Instant;
 
+#[derive(Clone, Copy)]
+struct StyleScheme {
+    color_enabled: bool,
+}
+impl StyleScheme {
+    fn no_color(self, text: String) -> StyledContent<String> {
```
Comment (hatoo) [inline src/printer.rs:19]: I think all `text: String` in `impl` can be rewritten as `text: &str` and a bunch of `.to_owned()` will disappear.

e.g `fn no_color(self, text: String) -> StyledContent<String>` to
`fn no_color(self, text: &str) -> StyledContent<&str>`

**Review by hatoo — CHANGES_REQUESTED** [review]
Thank you!

I left just one review!

**Comment by hatoo** [comment]
Clippy fail is OK. It's false positive.

**Comment by wjhoward** [comment]
Just pushed the `StyleScheme` String to &str change.
My plan is to next test this change on Ubuntu and Windows (already tested on MacOS).
I can then also update the `demo.gif`

**Comment by hatoo** [comment]
> I can then also update the demo.gif

This is really great, could you update `CHANGELOG.md` also?

**Comment by wjhoward** [comment]
Manual testing looked fine. The `demo.gif` is larger to contain the full summary view, hopefully that's okay for the README page.

**Review by hatoo — APPROVED** [review]

**Comment by hatoo** [comment]
LGTM. Thank you! 🎉

**Comment by wjhoward** [comment]
> LGTM. Thank you! 🎉

No problem, thank you!

**Pull request merged**